packages/element-modules/src/Node/ElementCore.Node.spec.js

```diff
@@ -0,0 +1,25 @@
+const ElementCore = require("@transmute/element-core");
```

Comment:
Looks like `ElementCore.Node.spec.js` and `ElementCore.React.spec.js ` are the same.

I assume it is to test that element-core works both in the browser, and in a node environment but don't see how the environment could be different between the two files

Reply:
yep

Reply:
yep as in, "yes they are the same and one can be deleted" or "yes they run in different environment"?

In the latter case, how?

Reply:
https://github.com/decentralized-identity/element/pull/45/files#diff-b9800c3aab96bca06c080a5c3f971140R14

using jest `--env` to set the environment and run each file.

Reply:
Ah yes, didn't see that mb.